AEM/optimum-habana-perf.md

```diff
@@ -44,26 +45,28 @@ pip install git+https://github.com/HabanaAI/DeepSpeed.git@1.19.0
 ```
 
 
-# Tensor quantization statistics measurement
-
-## Llama2
+# Tensor quantization statisics measurement
+Users just need to do once among different models with related world size values.  
+Users will get hqt_output after this step and it will be used for fp8 run.  
```

Comment:
Change to "After this step is complete the contents of the newly created hqt_optput directory can be used for fp8 runs.:

Reply:
use iGPT to improve the sentences accordingly.